test/integration/targets/ecs_tag/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+- set_fact:
```

Comment:
This virtual env shouldn't be needed, the requirements should be satisfied by the Ansible test container.  full_test.yml could be renamed to main.yml.

Reply:
removed old main and renamed full_test  I copied the ecs_cluster code.